src/frontend/dash/index.ts

```diff
@@ -12,66 +12,39 @@ window["openFullscreen"] = openFullscreen
 
 cyclePanel()
 setInterval(cyclePanel, 1000 * 60 * 1) // chnage panel every 1 minutes
+setInterval(regenCircles, 10) // chnage panel every 1 minutes
```

Comment:
```suggestion
setInterval(regenCircles, 1) // update circles every 1 minutes
```

Reply:
setInterval uses milliseconds for the second argument

Reply:
I didn't bother to change micah's comment-
it's meant to be 10 milliseconds cause it's not too long and not too short

Reply:
regen circle I think is the function that changes the circles' positions

Reply:
Thank you Zach, I should have checked.

Reply:
> Thank you Zach, I should have checked.

the value is fine right now

Reply:
Can the comment be updated accordingly then

Reply:
This function is also being called periodically on lines 67-69, every 10ms here, every 30s there.

Reply:
When it's being called on lines 67-69, it is also updating the circles, seeing what needs to be deleted.

Reply:
which means it also updates every 30 seconds to send a request to the server

Reply:
Sorry meant lines 64-66, every 60 seconds. They both call regenCircles, and have identical behavior.

Reply:
No, it's both lines 64-66 and 67-69.

Anyways, I removed lines 64-66 and 52 where regenCircles was being called.